mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -824,6 +843,44 @@ class LogisticRegressionSuite extends MLTest with DefaultReadWriteTest {
     assert(model6.coefficients ~= coefficientsExpected5 relTol 1E-3)
   }
 
+  test("SPARK-34448: binary logistic regression with intercept with features with small var") {
```

Comment:
this case fails in master:
![blr_center](https://user-images.githubusercontent.com/7322292/109479900-dd930080-7ab5-11eb-849c-e1332d810355.png)